test/views/coronavirus_local_restrictions_test.rb

```diff
@@ -0,0 +1,90 @@
+require "test_helper"
+require "gds_api/test_helpers/mapit"
+
+class CoronavirusLocalRestrictionTest < ActionView::TestCase
+  include GdsApi::TestHelpers::Mapit
+
+  test "renders postcode match tier 4" do
+    render_tier_results(4)
+
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_pretext")
+    assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_tier_label")
+    assert_includes rendered, area
```

Comment:
This area variable seems to appear by surprise. Perhaps it could be specified in an argument of the render method. That could be something like `render_view(tier: 4, area: "Tattooine")` and then you'd not need to find the area method.

Reply:
Thanks... I refactored!